juriscraper/opinions/united_states/state/lactapp_2.py

```diff
@@ -0,0 +1,147 @@
+        "Winn": "Eighth Judicial District Court for the Parish of Winn, Louisiana",
+        "Morehouse": "Fourth Judicial District Court for the Parish of Morehouse, Louisiana",
+        "Claiborne": "Second Judicial District Court for the Parish of Claiborne, Louisiana",
+        "Ouachita Monroe City Court": "Monroe City Court for the Parish of Ouachita, Louisiana",
+        "Bienville": "Second Judicial District Court for the Parish of Bienville, Louisiana",
+        "Madison": "Sixth Judicial District Court for the Parish of Madison, Louisiana",
+        "Red River": "Ninth Judicial District Court for the Parish of Red River, Louisiana",
+        "Tensas": "Sixth Judicial District Court for the Parish of Tensas, Louisiana",
+        "Jackson": "Second Judicial District Court for the Parish of Jackson, Louisiana",
+        "Ouachita OWC District 1-E": "Office of Workers' Compensation District 1-E for the Parish of Ouachita, Louisiana",
+        "Caddo OWC District 1-W": "Office of Workers' Compensation District 1-W for the Parish of Caddo, Louisiana",
+        "Caldwell": "Thirty-Seventh Judicial District Court for the Parish of Caldwell, Louisiana",
+        "West Carroll": "Fifth Judicial District Court for the Parish of West Carroll, Louisiana",
+        "East Carroll": "Sixth Judicial District Court for the Parish of East Carroll, Louisiana",
+        "Caddo Juvenile Court": "Juvenile Court for the Parish of Caddo, Louisiana",
+        "Caddo Shreveport City Court": "Shreveport City Court for the Parish of Caddo, Louisiana",
+        "DeSoto OWC District 1-W": "Office of Workers' Compensation District 1-W for the Parish of DeSoto, Louisiana",
+        "Lincoln Ruston City Court": "Ruston City Court for the Parish of Lincoln, Louisiana",
+        "Ouachita West Monroe City Court": "West Monroe City Court for the Parish of Ouachita, Louisiana",
+        "OUACHITA Monroe City Court": "Monroe City Court for the Parish of Ouachita, Louisiana",
+        "Franklin OWC District 1-E": "Office of Workers' Compensation District 1-E for the Parish of Franklin, Louisiana",
+        "Minden City Court Webster": "Minden City Court for the Parish of Webster, Louisiana",
+        "Morehouse Bastrop City Court": "Bastrop City Court for the Parish of Morehouse, Louisiana",
+        "Morehouse OWC District 1-E": "Office of Workers' Compensation District 1-E for the Parish of Morehouse, Louisiana",
+        "Webster Minden City Court": "Minden City Court for the Parish of Webster, Louisiana",
+        "Winn OWC District 2": "Office of Workers' Compensation District 2 for the Parish of Winn, Louisiana",
+    }
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        self.base_url = "https://www.la2nd.org/opinions/"
+        self.year = datetime.now().year
+        self.url = f"{self.base_url}?opinion_year={self.year}"
```

Comment:
I've been trained by @grossir to use better tools around making URLs 

What about something like this 

        params = {"opinion_year": self.year}
        self.url = urljoin(base_url, f"?{urlencode(params)}")